app/application.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+var root = 'http://localhost:4000/api';
```

Comment:
How about using `const` for these https://github.com/thoughtbot/guides/blob/master/style/javascript/README.md